contents/tutorials/webflow-ab-tests.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "How to run A/B tests in Webflow with PostHog"
+date: 2023-01-26
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-2.png
+topics: ['experimentation', 'feature flags']
+---
+
+Optimizing your marketing site often requires testing small changes against each other, also known as A/B tests. Getting the best site possible requires tweaking, experimenting, and tracking results.
+
+Webflow, as a no-code site builder, makes it easy to make these changes. PostHog, with its built-in experimentation suite, makes it easy to set up and track A/B tests and experiments. 
+
+This tutorial shows how to set up A/B tests with Webflow and PostHog to get the most out of your marketing site.
+
+> Thanks to Julia at [Mayan](https://www.mayan.co/) for inspiring this tutorial.
```

Comment:
Is it worth embedding her video here?

Reply:
I think this is better because there are some steps she misses, and I will record another version.